packages/shared/src/utils/helpers/__tests__/durations.ts

```diff
@@ -41,11 +42,21 @@ describe('buildDurationConfig', () => {
     };
 
     it('Returns correct value when durations is not passed', () => {
-        expect(Duration.buildDurationConfig(contract)).to.eql(durations);
+        expect(
+            Duration.buildDurationConfig(
+                contract,
+                durations ?? { min_max: { spot: {}, forward: {} }, units_display: {} }
+            )
+        ).to.eql(durations);
     });
 
     it('Returns correct value when durations passed', () => {
-        expect(Duration.buildDurationConfig(contract, durations)).to.eql(durations);
+        expect(
+            Duration.buildDurationConfig(
+                contract,
+                durations ?? { min_max: { spot: {}, forward: {} }, units_display: {} }
```

Comment:
maybe it would be more clear to use here '||" instead of '??'
because '??' means, that the value can be 0

Reply:
@jim-deriv could you please tell which TS error or warning made you add this extra check `?? { min_max: { spot: {}, forward: {} }, units_display: {} }`  across the files?🙂 Should be avoidable since you have `durations` typed with `TDurations` above.